posts/2020/03/ibis-1.3-notes.ipynb

```diff
@@ -0,0 +1,1297 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Ibis framework 1.3 was just released, after 8 months of development work with 104 new commits from 16 distinct contributors. What is new? In this blog post we will check some important highlighted features about this new version!\n",
+    "\n",
+    "First, if you are new to Ibis framework world, you can check this [blog-post](https://labs.quansight.org/blog/2019/07/ibis-python-data-analysis-productivity-framework/) wrote last year with some introductory information about that.\n",
+    "\n",
+    "Some hightlighted features of this new version are:\n",
+    "\n",
+    "* Renaming `mapd` backend to `omniscidb`\n",
+    "* Added new backends and geospatial methods to the documentation\n",
+    "* Improvement of geospatial support\n",
+    "* Addition of `PySpark` backend\n",
+    "* Addition of `JSON`, `JSONB` and `UUID` data types\n",
+    "* Support for `Python 3.5` dropped.\n",
+    "* Initial support for `Python 3.8` added.\n",
+    "\n",
+    "This blog post is divided into different notes section, such as:\n",
+    "\n",
+    "* OmniSciDB\n",
+    "* PostgreSQL\n",
+    "* PySpark\n",
+    "* Geospatial support\n",
+    "* Python versions support"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ibis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### OmniSciDB notes\n",
+    "\n",
+    "Remember, `mapd` backend now is `omniscidb`! \n",
+    "\n",
+    "Another important feature is that now you can define if the connection is `IPC` and you can also specify the `GPU` device ID you want to use it (if you have a NVIDIA card, supported by `cudf`)."
```

Comment:
I don't think the rename is a feature, so I wouldn't call this "another feature"

Also, may be explain briefly what's IPC?